stream-chat-android-test/src/main/java/io/getstream/chat/android/test/TestCall.kt

```diff
@@ -1,10 +1,10 @@
-package io.getstream.chat.android.livedata.utils
+package io.getstream.chat.android.test
 
 import io.getstream.chat.android.client.call.Call
 import io.getstream.chat.android.client.utils.Result
 
-internal class TestCall<T : Any>(val result: Result<T>) : Call<T> {
-    var cancelled = false
+public class TestCall<T : Any>(public val result: Result<T>) : Call<T> {
```

Comment:
Sould it be annotated as `@InternalStreamChatApi` ?

Reply:
I figured that since we're only including this module as `testImplementation`, it's easier to just have everything in it be public. But if you think people might start using this module themselves (?), then we could add annotations to scare them away from doing so.

Reply:
Really, we just shouldn't be publishing this module, but Jitpack... Not sure if they allow us to configure this kind of thing, to exclude this module from publication.

Reply:
I am not sure they provide any configuration to exclude this, I wouldn't take care about it for now, only limit the things we think are not needed for third-party devs